ompi/mca/osc/ucx/osc_ucx_comm.c

```diff
@@ -358,7 +358,9 @@ static inline int get_dynamic_win_info(uint64_t remote_addr,
     free(temp_buf);
 
     /* unlock the dynamic lock */
-    return ompi_osc_ucx_dynamic_unlock(module, target);
+    ret_unlock = ompi_osc_ucx_dynamic_unlock(module, target);
+    /* in case of error return error code ignoring unlock result */
```

Comment:
imo better rephrase somehow, e.g. `ignore unlock result in case of error`

Reply:
fixed